UnofficialSteamAuthenticator/App.xaml.cs

```diff
@@ -14,9 +14,9 @@ namespace UnofficialSteamAuthenticator
     /// <summary>
     /// Provides application-specific behavior to supplement the default Application class.
     /// </summary>
-    public sealed partial class App : Application
```

Comment:
Displayed wrongly by GitHub... I just removed the inheritance because it's a super type reference and already specified in the App.xaml.